docs/guidelines/comments.rst

```diff
@@ -0,0 +1,125 @@
+Comments
+========
+
+.. note::
+
+    If peppering your code with lots of comments is good, then having zillions of comments in your code must be great,
+    right? Not quite. Comments in code are a double edged sword, use them to describe what's being done.
+
+
+It doesn't make sense to comment every step you do in your program. Comments should discuss why something is done,
+its purpose and its goal. The code already shows how it is done, so commenting on this is redundant.
+
+Section Comments
+----------------
+
+The first separation style is *section comment*. This uses the full 120 width span ideally placed to separate large
+chunks of logic::
+
+    // #####################################################################################################################
+    // #NAME#
+
+The second available style separates smaller code blocks using 60 only lines::
+
+    // #########################################################
+    // #NAME#
+
+
+Inline Comments
+---------------
+
+When using comments inline, make use of the appropriate formats:
+
+- ``{# ... #}`` or ``{% comment %} ... {% endcomment %}`` for Django templates and **never** ``<!-- ... -->``
+- ``// ...`` and ``/* ... */`` for SASS and JavaScript
+
+Notes
+*****
+
+We also use several comment helpers which, if you configured them in your editor, add additional highlighting to your
+code:
+
+| **FIXME:**
+| to annotate problems with the code
+|
+
+.. code-block:: javascript
+
+    function Calculator() {
+        // FIXME: shouldn't use a global here
+        total = 0;
+        ...
+    }
+
+
+| **TODO:**
+| to annotate solutions to problems with the code
+|
+
+.. code-block:: javascript
+
+    function Calculator() {
+        // TODO: total should be configurable by an options param
+        this.total = 0;
+        ...
+    }
+
+| **INFO:**
```

Comment:
we could easily remove this, it's kind of unnecessary, since that's an inherent quality of the comments – they provide information
if nothing changes if we omit the INFO keyword - does it make sense to have it at all?